arlo-client/src/setupProxy.js

```diff
@@ -20,9 +20,9 @@ module.exports = function(app) {
   app.use(proxy('/election/*/sample-sizes', { target }))
   app.use(proxy('/election/*/contest', { target }))
   app.use(proxy('/election/*/round', { target }))
-  app.use(proxy('/election/*/jurisdiction', { target }))
```

Comment:
This path is used in `useJurisdictions` for the status box: https://github.com/votingworks/arlo/blob/389526c5dc588da4808c13132574cecc2672de4d/arlo-client/src/components/Audit/useJurisdictions.ts#L49

So this breaks on the AA view.